pkg/kritis/kubectl/plugins/resolve/resolve_test.go

```diff
@@ -223,6 +227,43 @@ func Test_recursiveReplaceImage(t *testing.T) {
 	}
 }
 
+func Test_Execute(t *testing.T) {
+	testYaml := `apiVersion: v1
+kind: Pod
+metadata:
+  name: test
+spec:
+  containers:
+  - name: debian
+    image: %s`
+	initial := fmt.Sprintf(testYaml, "gcr.io/kaniko-project/executor:v0.1.0")
+	expected := fmt.Sprintf(testYaml, "gcr.io/kaniko-project/executor@sha256:501056bf52f3a96f151ccbeb028715330d5d5aa6647e7572ce6c6c55f91ab374")
```

Comment:
do you want to mock this out and use a fixed value?

Reply:
I sort of treated this like an integration test and chose an image that we probably won't change just to make sure the entire flow worked. Would it make more sense to do that somewhere else?

Reply:
Yeah we dont want to rely on already existing image also used for other purpose.
If you want to add an integration test, we can create a new image specific for this purpose and push it to a project dedicated for integration testing.
I am going to create "kritis-int-test" project and set it up for the team.
You can push a new image to this project. 
Does that make sense?

Reply:
Yup sounds good!